tests/sidetrail/DEBUGGING.md

```diff
@@ -84,15 +84,15 @@ If there are any functions in the warning that are not in that list, then one of
 
 When Sidetrail analyzes a program, it needs to know how many times a loop will execute.
 If Sidetrail encounters a loop, and can't tell how many times it will execute, it may give a spurious counterexample.
-You can fix this using the `S2N_INVARIENT` macro.
+You can fix this using the `S2N_INVARIANT` macro.
```

Comment:
Unfortunately, it's actually spelled `S2N_INVARIENT` in the code:
https://github.com/awslabs/s2n/blob/e6eaf873543cf27583ac96e006064f0582ce3a97/utils/s2n_annotations.h#L19

As part of this PR, though, feel free to fix that :smile: !

Reply:
:o! Thank you @camshaft. I should have checked.

Reply:
Done

Reply:
I think there's even more:

https://github.com/awslabs/s2n/blob/e6eaf873543cf27583ac96e006064f0582ce3a97/tests/sidetrail/working/stubs/s2n_annotations.h#L22

Reply:
Indeed! Thanks. Hopefully, this time I squashed them all :)